account/tests.py

```diff
@@ -0,0 +1,163 @@
+from django.contrib.auth import get_user_model
+from .models.profile import Profile, UserType
+from django.core.exceptions import ValidationError
+import pytest
+
+
+@pytest.mark.django_db
+class TestProfileModel:
+
+    @pytest.fixture
+    def user(self):
+        return get_user_model().objects.create_user(
+            username='testuser',
+            password='testpass'
+        )
+
+    @pytest.fixture
+    def saved_profile(self, user):
+        profile = Profile.objects.create(
+            user_id=user,
+            user_type=UserType.Professional,
+            phone_number='0000000000',
+            country='testcountry',
+            city='testcity',
+            address='testaddress'
+        )
+        return profile
+
+    def test_get_profile(self, saved_profile):
+        profile = Profile.objects.get(pk=saved_profile.pk)
+        assert profile == saved_profile
+
+    def test_delete_profile(self, saved_profile):
+        profile = saved_profile
+        profile.delete()
+        with pytest.raises(Profile.DoesNotExist):
+            Profile.objects.get(pk=saved_profile.pk)
```

Comment:
Test should end with assertion not throwing Exceptions
You could assert Exception has raised